src/main/java/uk/gov/companieshouse/api/strikeoffobjections/controller/ObjectionController.java

```diff
@@ -439,4 +442,49 @@ public ResponseEntity deleteAttachment(
             );
         }
     }
+
+    @GetMapping("/{objectionId}/attachments/{attachmentId}/download")
+    public ResponseEntity<Void> downloadAttachment(@PathVariable String companyNumber,
+                                                   @PathVariable String objectionId,
+                                                   @PathVariable String attachmentId,
+                                                   @RequestHeader(value = ERIC_REQUEST_ID_HEADER) String requestId,
+                                                   HttpServletResponse response) {
+        Map<String, Object> logMap = new HashMap<>();
+        logMap.put(LOG_COMPANY_NUMBER_KEY, companyNumber);
+        logMap.put(LOG_OBJECTION_ID_KEY, objectionId);
+
+        try{
+            apiLogger.infoContext(
+                    requestId,
+                    "GET /{objectionId}/attachments/{attachmentId}/download request received",
+                    logMap
+            );
+            FileTransferApiClientResponse downloadServiceResult = objectionService.downloadAttachment(
+                    requestId, objectionId, attachmentId, response);
+            return ResponseEntity.status(downloadServiceResult.getHttpStatus()).build();
+        } catch (HttpClientErrorException | HttpServerErrorException e) {
+            apiLogger.errorContext(
+                    requestId,
+                    DOWNLOAD_ERROR,
+                    e,
+                    logMap
+            );
+            return ResponseEntity.status(e.getStatusCode()).build();
+        } catch (ServiceException e) {
```

Comment:
Need a unit test to cover the service exception being thrown

Reply:
Added a test.